src/utils/command.js

```diff
@@ -25,16 +25,28 @@ class BaseCommand extends Command {
   }
   // Initialize context
   async init(_projectRoot) {
-    const projectRoot = findRoot(_projectRoot || process.cwd()) // if calling programmatically, can use a supplied root, else in normal CLI context it just uses process.cwd()
+    const cwd = argv.cwd || process.cwd()
+    const projectRoot = findRoot(_projectRoot || cwd) // if calling programmatically, can use a supplied root, else in normal CLI context it just uses process.cwd()
     // Grab netlify API token
     const authViaFlag = getAuthArg(argv)
 
     const [token] = this.getConfigToken(authViaFlag)
-    // Get site config from netlify.toml
-    const configPath = getConfigPath(projectRoot)
-    // TODO: https://github.com/request/caseless to handle key casing issues
-    const config = readConfig(configPath)
 
+    // Read new netlify.toml/yml/json
+    let configPath
+    let config = {}
+    try {
+      configPath = await getConfigPath(argv.config, cwd)
+      config = await resolveConfig(configPath, {
+        cwd: cwd,
+        context: argv.context
+      })
+    } catch (err) {
+      // Suppress config not found error for CLI. @TODO Revisit
+      if (err.message.indexOf('No netlify configuration file was found') === -1) {
```

Comment:
Why suppress this error?

Reply:
Not every site needs netlify config file. They might have build settings set in the UI

The @netlify/build config package was throwing hard errors when config was missing.

Reply:
It might be updated in build? @ehmicky did you update how config handles this?

Reply:
I see.

Reply:
ref: https://github.com/netlify/build/issues/497#issuecomment-557100286

Reply:
Yes this `try`/`catch` should not be needed anymore. Missing configuration files do not throw anymore.